preprocessing/jupyter-structured/jupyter-generate-triplets.py

```diff
@@ -0,0 +1,529 @@
+<!DOCTYPE html>
```

Comment:
is this supposed to be a notebook?

Reply:
Sorry, it was an unintended mistake during copy/paste. I have updated it now.